### ISSUE TITLE
Not legal; breaks Operator license

### ISSUE THREAD

**Issue by thundernixon** [HamishBrindle/operator-mono-lig#2]
Operator is not an open source font. You can't just edit it and release it for free. Sorry. :/

There are some great open source code fonts, though! See [Fira Code](https://github.com/tonsky/FiraCode) for one with ligatures.

**Comment by kiliman** [comment]
Yes. @HamishBrindle took the output of my project and uploaded the generated fonts.

https://github.com/kiliman/operator-mono-lig

My project will patch your existing fonts (which you should have a license for) to add the ligatures. I did it that way because it was not a free font and could not be redistributed.

@HamishBrindle put projects like mine in jeopardy by simply uploading the complete font. You were obviously able to build the patched font, so why don't you take this down and simply point others to my repo.
